src/components/member-profile/index.js

```diff
@@ -351,12 +359,22 @@ const Profile = (props) => {
   };
 
   return (
-    <>
+    <div
+      role="button"
+      tabIndex={0}
+      onKeyDown={(e) => {
+        if (e.altKey) {
+          setIsOptionKey(true);
+        }
+      }}
```

Comment:
This will be removed when this issue is closed 
- https://github.com/Real-Dev-Squad/website-members/issues/420